game-core/src/main/java/games/strategy/engine/framework/network/ui/SetMapClientAction.java

```diff
@@ -17,29 +16,28 @@
 
   final List<String> availableGames;
   private final Component parent;
-  private final IClientMessenger clientMessenger;
+  private final IServerStartupRemote serverStartupRemote;
 
   public SetMapClientAction(
       final Component parent,
-      final IClientMessenger clientMessenger,
+      final IServerStartupRemote serverStartupRemote,
       final List<String> availableGames) {
     super("Change Game To");
     this.parent = JOptionPane.getFrameForComponent(parent);
-    this.clientMessenger = clientMessenger;
+    this.serverStartupRemote = serverStartupRemote;
     this.availableGames = availableGames;
     Collections.sort(this.availableGames);
   }
 
   @Override
   public void actionPerformed(final ActionEvent e) {
-    final INode serverNode = clientMessenger.getServerNode();
     final DefaultComboBoxModel<String> model = new DefaultComboBoxModel<>();
     final JComboBox<String> combo = new JComboBox<>(model);
     model.addElement("");
     for (final String game : availableGames) {
       model.addElement(game);
     }
-    if (serverNode == null || model.getSize() <= 1) {
+    if (model.getSize() <= 1) {
```

Comment:
This is a change I'm not sure is 100% safe, but it worked no problem so I didn't want to pass clientMessenger as well just for this null check